src/components/Accordion/AccordionSection.js

```diff
@@ -0,0 +1,102 @@
+import React, { useRef, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import styles from './AccordionSection.module.css'
+import { useAccordionContext } from './useAccordionContext.js'
+import { codes } from '../../helpers/constants.js'
+
+export const AccordionSection = ({ children, title, renderToggle, index }) => {
+  const {
+    focusRef,
+    selected,
+    expandedAll,
+    onToggle,
+    onNavigation,
+    id,
+  } = useAccordionContext()
+  const sectionId = `${id}-${index}-section`
+  const labelId = `${id}-${index}-label`
+  const expanded = expandedAll[index]
+  const labelRef = useRef()
+
+  useEffect(() => {
+    if (index === selected[0] && labelRef.current) {
+      labelRef.current.focus()
+    }
+  }, [index, selected])
+
+  const onKeyDown = (e) => {
+    switch (e.keyCode) {
+      case codes.SPACE:
+      case codes.RETURN:
+        e.preventDefault()
+        onToggle && onToggle(index)
+        break
+      case codes.DOWN:
+        e.preventDefault()
+        onNavigation(codes.DOWN)
+        break
+      case codes.UP:
+        e.preventDefault()
+        onNavigation(codes.UP)
+        break
+      case codes.HOME:
+        e.preventDefault()
+        onNavigation(codes.HOME)
+        break
+      case codes.END:
+        e.preventDefault()
+        onNavigation(codes.END)
+        break
+      default:
+    }
+  }
+
+  const onFocus = () => {
+    focusRef.current = index
+  }
+
+  const onBlur = () => {
+    focusRef.current = null
+  }
+
+  const onClick = () => {
+    onToggle && onToggle(index)
+  }
+
+  return (
+    <>
+      <div
+        role="button"
+        aria-expanded={expanded}
+        aria-controls={sectionId}
+        id={labelId}
+        tabIndex={0}
+        className={styles.Label}
```

Comment:
I'm thinking about what needs we have for `Nora` and was wondering if we could make the style of the `title` customizable? Right now, Nora section headers have pretty particular styling

![image](https://user-images.githubusercontent.com/12839832/75809253-55143f80-5d3d-11ea-91c8-b67c47d8234c.png)